POS_Image-RPi-Bootstrap/rpi-bootstrap-7.0.0/images.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2019 SUSE LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+# In typical case the original root filesystem will be replaced byt the downloaded image
+# this disables resizing to save time
+rpm -e dracut-kiwi-oem-repart
```

Comment:
Why are we adding this in config.xml and remove here?

Reply:
Kiwi fails if the image type is oem and this package is missing in config.xml.

Reply:
Can't this also be done by [oemresize](https://osinside.github.io/kiwi/building_images/build_expandable_disk.html#oem-customize)

```xml
<oemconfig>
  <oem-resize>False</oem-resize>
</oemconfig>
```